src/gt4py/next/ffront/itir_makers.py

```diff
@@ -271,5 +271,29 @@ def shift_(offset, value=None):
     return call_(call_("shift")(*args))
 
 
+def neighbors_(offset, it):
+    offset = ensure_offset(offset)  # TODO this function should not exist?
```

Comment:
```suggestion
    offset = ensure_offset(offset)
```
I think it should exist here, because itir_makers provides convenience functions, i.e. should automatically wrap in `OffsetLiteral` when needed.

Maybe rename to `as_offset` or `to_offset`.